Classes/NanoKontrol2.sc

```diff
@@ -150,15 +150,15 @@ NK2Button : NK2Controller {
     }
 
     onPress_ {|func|
-        MIDIdef.cc(key, {|val|
+        MIDIdef.cc(key ++ \On, {|val|
```

Comment:
This will promote the key to a `String`, please wrap in .asSymbol like this: 
`(key ++ "_on").asSymbol`